app/web/components/LanguagePickerSettings.tsx

```diff
@@ -0,0 +1,149 @@
+import {
+  Link as MuiLink,
+  Typography,
+  useMediaQuery,
+  useTheme,
+} from "@mui/material";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import { Empty } from "google-protobuf/google/protobuf/empty_pb";
+import { RpcError } from "grpc-web";
+import { useTranslation } from "i18n";
+import { LANGUAGE_MAP } from "i18n/constants";
+import { AUTH, GLOBAL } from "i18n/namespaces";
+import { useForm } from "react-hook-form";
+import { useMutation } from "react-query";
+import { service } from "service";
+
+import useChangeDetailsFormStyles from "../features/auth/useChangeDetailsFormStyles";
+import LanguagePickerSelect from "./LanguagePickerSelect";
+
+// TODO: Update to correct link
+const COMMUNITY_BUILDER_FORM_LINK =
+  "https://couchers.org/community-builder-form";
```

Comment:
Is this still a to-do?

Reply:
@laurabotel this link says community builder link though? Or is github showing the wrong thing?

@jesseallhands would know where we should link to for people to help with language translation though!